src/query/sql/src/planner/binder/ddl/data_mask.rs

```diff
@@ -33,6 +34,16 @@ impl Binder {
             policy,
         } = stmt;
 
+        // check if input type match to the return type
+        let return_type = policy.return_type.to_string().to_lowercase();
```

Comment:
Need a logic test for this

Reply:
see: tests/sqllogictests/suites/ee/05_ee_ddl/05_0001_ddl_data_mask in this pr